src/core_plugins/kibana/public/home/components/tutorial/replace_template_strings.js

```diff
@@ -31,5 +31,9 @@ export function replaceTemplateStrings(text, params = {}) {
     params: params
   };
   Mustache.parse(text, TEMPLATE_TAGS);
+  // do not html escape output
+  Mustache.escape = function (value) {
```

Comment:
This is modifying the global `Mustache` escape function... What did this PR introduce that makes us turn off the HTML escaping?

Reply:
The other outputs of mustache are getting run thru markdown-it. That seems to un-escape the escaped output of mustache. For this PR, it displays the results of mustache without running it thru markdown-it. The escaped html breaks URL links